operator_ui/package.json

```diff
@@ -8,17 +8,21 @@
   },
   "scripts": {
     "preinstall": "node -e \"if(process.env.npm_execpath.indexOf('yarn') === -1) throw new Error('Please use yarn to install dependencies\\n\\n')\"",
+    "prestart": "yarn generate",
     "start": "webpack serve --config webpack.dev.js",
     "serve": "serve dist -p 3000 --config webpack.prod.js",
     "test": "jest",
     "test:ci": "yarn test --reporters jest-silent-reporter --maxWorkers=50%",
     "watch": "yarn test --watchAll --notify",
     "clean": "tsc -b --clean && rimraf -rf artifacts tmp dist",
     "build": "NODE_ENV=production webpack --config webpack.prod.js",
+    "presetup": "yarn generate",
```

Comment:
Generate the GQL types on before setup.

Setup is used when running CI tests and building operator ui